### ISSUE TITLE
VSCode 1.94.0 ANSI colors in Debug Console not working

### ISSUE THREAD

**Issue by nathan831** [onsi/ginkgo#1466]
Hi,
After updating VSCode to 1.94.0, ANSI colors stopped working in the debug console. After a quick search, I found the following problem, which can be reproduced exactly the same way for ginko. https://github.com/Dart-Code/Dart-Code/issues/5302

Thanks in advance
Ginkgo version used v2.20.2
Related to https://github.com/microsoft/vscode/pull/227729

**Comment by onsi** [comment]
hey @nathan831 - ginkgo doesn't ship with a vscode extension (there's a basic one that does snippets - but nothing more integrated than that).  So I'm not sure how to help here...

**Comment by alanosullivancs** [comment]
hi @onsi this is a general VS Code bug:

V 194

![vs_code_194 2 color_debug_console_bug](https://github.com/user-attachments/assets/155aaa45-8ae0-48d0-bebc-897705ebc9a6)

V 193

![image](https://github.com/user-attachments/assets/936c1dc8-11ae-4385-8923-d05e342e83e1)

**Comment by onsi** [comment]
ok.. understood.  I assume they'll fix it in VSCode eventually?  Is there anything you're wanting to see changed in Ginkgo to fix this?

**Comment by alanosullivancs** [comment]
Nope, users of VS Code need to apply this fix https://github.com/microsoft/vscode-js-debug/?tab=readme-ov-file#nightly-extension until the change is permanently fixed
